dds_web/api/project.py

```diff
@@ -486,8 +485,11 @@ def delete_project(self, project: models.Project, current_time: datetime.datetim
             )
             self.rm_project_user_keys(project=project)
 
+            # Only mark as inactive after all deletion operations succeed
+            project.is_active = False
```

Comment:
Would be good to add a test that mocks `delete_project_contents`, and one that mocks `rm_project_user_keys`; They should fail and we want to test that the `project.is_active` is still `True`